draft-ietf-webpush-protocol.xml

```diff
@@ -305,7 +305,22 @@
         </t>
       </section>
     </section>
-
+    <section title="Connecting to the Push Service">
+      <t>
+        The push service shares the same default port number (443/TCP) with HTTPS, but MAY
+        also advertise the IANA allocated TCP port 1001 using HTTP alternative services
+        <xref target="I-D.ietf-httpbis-alt-svc"/>:
+      </t>
+      <t>
+        The configuration of this port can potentially be optimized for better timeout
+        characteristics for push notifications than port 443.
+      </t>
+      <t>
+        NAT(s) MUST comply with REQ-5 in <xref target="RFC7231"/> which requires that
+        "the value of the 'established connection idle-timeout' MUST NOT be less than
+        2 hours 4 minutes".

```

Comment:
Better to be pragmatic then. I spent some time reviewing how other drafts and/or RFC(s) referenced RFC 5382 [BEHAVE] for similar scenarios and found the closest match in Section 7 of https://www.rfc-editor.org/rfc/rfc5383.txt which used a MUST.